documentation/src/docs/pravega-concepts.md

```diff
@@ -93,6 +114,13 @@ Stream Segments. Event Routing Keys are hashed to form a "key space". The key
 space is then divided into a number of partitions, corresponding to the number
 of Stream Segments. Consistent hashing determines of Events to Stream Segments.
 
+### Stream Segments and Connection pooling
+
+When an Event is written to the Stream by the Pravega Client it is written into one of the Stream Segments based on the Event Routing Key. These Segments which are a part of Segment Containers are managed by the different [Segment Store](segment-store-service.md) Service instances.
```

Comment:
1. You use "written" twice in this sentence making it hard to read. Please rephrase.

2. `These Segments which are part of Segment Containers.` This sounds weird, please rephrase or remove the Container part.

Reply:
@andreipaduroiu Accepted suggestion